SECURITY.md

```diff
@@ -0,0 +1,249 @@
+* **Run security analysis tools** before submitting pull requests
+* **Report security concerns** privately through the proper channels
+
+## Input Validation
+
+VHDLTest implements input validation for:
+
+### Configuration Files (YAML)
+
+* YAML configuration files are parsed using YamlDotNet
+* Schema validation ensures required fields are present
+* File paths are validated to prevent directory traversal attacks
+* Malformed YAML files are rejected with clear error messages
+
+### Command-Line Arguments
+
+* All command-line arguments are validated before use
+* File paths are sanitized to prevent path traversal
+* Simulator names are validated against a whitelist
+* Invalid arguments result in clear error messages and safe exit
+
+### VHDL Files
+
+* File existence and readability are verified before processing
+* File paths are validated to prevent directory traversal
+* Large files are handled appropriately to prevent resource exhaustion
+
+### External Tool Invocation
+
+* Simulator paths are validated before invocation
+* Command arguments are properly escaped to prevent injection attacks
+* Process execution is monitored and constrained
+
+## Security Tools Used
+
+VHDLTest employs multiple security tools and practices:
+
+### Static Analysis
+
+* **Microsoft.CodeAnalysis.NetAnalyzers**: Provides security analysis during compilation
+* **SonarCloud**: Continuous security analysis and vulnerability detection
+* **CodeQL**: Automated security scanning in CI/CD pipeline
+* **Dependabot**: Automated dependency vulnerability scanning and updates
+
+### Code Quality
+
+* **EditorConfig**: Enforces consistent code formatting to reduce errors
+* **Warnings as Errors**: All compiler warnings are treated as errors
+* **Nullable Reference Types**: Enabled to prevent null reference exceptions
+
+### CI/CD Security
+
+* **SBOM Generation**: Software Bill of Materials for supply chain security
+* **Dependency Scanning**: Weekly automated checks for vulnerable dependencies
+* **Build Isolation**: Builds run in isolated, ephemeral environments
+* **Code Signing**: Releases are signed for authenticity verification
+
+### Testing
+
+* **Comprehensive Unit Tests**: Validate security-critical code paths
+* **Code Coverage**: Monitored to ensure test completeness
+* **Integration Tests**: Test interactions with external simulators safely
+
+## Responsible Disclosure
+
+We follow responsible disclosure practices:
+
+* **Coordinated Disclosure**: We work with security researchers to coordinate public disclosure
+* **Reasonable Timeline**: We aim to fix and disclose vulnerabilities within 90 days
+* **Private Discussion**: Vulnerabilities are discussed privately until patched
+* **Public Advisory**: After fixes are released, we publish detailed security advisories
+* **CVE Assignment**: We request CVE identifiers for significant vulnerabilities when appropriate
+
+### Our Commitment
```

Comment:
As this is just a stand-alone tool I think we can omit the commitment section - there is no user data or services to disrupt.